_sandbox/stereofy/main.cpp

```diff
@@ -81,7 +81,7 @@ int main(int argc, char* argv[])
 	// open the soundfile
 	if(!(fileIn = sf_open (inputFilename.c_str(), SFM_READ, &sfinfoIn)))
 	{
-		std::cout << "stereofy: failed to open file" << sf_strerror(fileIn) << std::endl;
+		std::cout << "stereofy: failed to open file: " << sf_strerror(fileIn) << std::endl;
```

Comment:
will we recompile?

Reply:
yes we will. better to do it manually in the processing machines because the deploy scripts skips recompilation when binary existing